mantle/platform/conf/conf.go

```diff
@@ -377,21 +337,23 @@ func (c *Conf) MergeV34exp(newConfig v34exptypes.Config) {
 	c.ignitionV34exp = &mergeConfig
 }
 
-// Merge all configs into a V3.4 config
-// configs must be of type V34exp to merge
-func MergeAllV34exp(confObjs []*Conf) (*UserData, error) {
-	config := Conf{}
-	config.ignitionV34exp = &v34exptypes.Config{
-		Ignition: v34exptypes.Ignition{
-			Version: "3.4.0-experimental",
+// Merge all configs into a V3.3 config
+func MergeAllConfigs(userdataObjs []*Conf) (*UserData, error) {
```

Comment:
Nit: the name `userdataObjs` is now misleading; maybe switch back to `confObjs`?